zeebe/qa/util/src/main/java/io/camunda/zeebe/qa/util/cluster/TestSpringApplication.java

```diff
@@ -27,7 +28,7 @@
 
 abstract class TestSpringApplication<T extends TestSpringApplication<T>>
     implements TestApplication<T> {
-  private final Class<?> springApplication;
+  private final List<Class<?>> springApplications;
```

Comment:
😄 This is also part of https://github.com/camunda/camunda/pull/18964

Reply:
I knew that, but I wanted to make the PR green 😅 Sure, I can review the other PR, then this part should become obsolete.